src/textual/_ansi_theme.py

```diff
@@ -0,0 +1,53 @@
+from rich.terminal_theme import TerminalTheme
+
+MONOKAI_DARK = TerminalTheme(
+    (12, 12, 12),
+    (217, 217, 217),
+    [
+        (26, 26, 26),
+        (244, 0, 95),
+        (152, 224, 36),
+        (253, 151, 31),
+        (157, 101, 255),
+        (244, 0, 95),
+        (88, 209, 235),
+        (196, 197, 181),
+        (98, 94, 76),
+    ],
+    [
+        (244, 0, 95),
+        (152, 224, 36),
+        (224, 213, 97),
+        (157, 101, 255),
+        (244, 0, 95),
+        (88, 209, 235),
+        (246, 246, 239),
+    ],
+)
+
+MONOKAI_LIGHT = TerminalTheme(
```

Comment:
Is this MONOKAI with the foreground and background swapped? Monokai doesn't really have a light variant AFAIK. I think we should pick a dedicated light theme.

Let's pick a theme from https://iterm2colorschemes.com/

Then use this tool to export it:

https://github.com/jeremyschulman/iterm2-colors2rgb